### PR TITLE
added endpoints

### PR DESCRIPTION
**Pull request by KrisPlanka**

### CODE REVIEW

File: src/controllers/phones.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const phonesService = require('../services/phones');
+
+const getAll = (req, res) => {
+  const phones = phonesService.getAll();
```
Comment (VladimirEdelmann) [inline src/controllers/phones.js:6]: getAll returns a promise, so you need to handle it with then method
```js
const getAll = (req, res) => {
  phonesService
    .getAll()
    .then((phones) => {
      res.send(phones);
    })
    .catch((err) => {
      console.log(err);
    });
};
```

File: src/controllers/phones.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  res.send(phones);
+};
+
+const getOne = (req, res) => {
+  const { phoneId } = req.params;
+  const foundPhone = phonesService.findById(phoneId);
+
```
Comment (VladimirEdelmann) [inline src/controllers/phones.js:14]: same here
```js
const getOne = (req, res) => {
  const { phoneId } = req.params;

  phonesService
    .findById(phoneId)
    .then((foundPhone) => {
      if (!foundPhone) {
        res.sendStatus(404);

        return;
      }

      res.send(foundPhone);

      return foundPhone;
    })
    .catch((err) => {
      console.error(err);
      res.sendStatus(500);
    });
};
```

**Review by VladimirEdelmann — CHANGES_REQUESTED** [review]
Nice work!

File: src/createServer.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const { router:  phonesRouter } = require('./src/routes/phones');
+
+const { phonesService } = require('./src/services/phones')
+
+
+function createServer() {
+  const app = express();
+
+  app.use('/phones', phonesRouter);
+  phonesService.reset();
```
[the remark this answers is not in the document]
Comment (KrisPlanka) [reply src/createServer.js:14]: fixed

**Review by VladimirEdelmann — APPROVED** [review]
Great!

**Pull request merged**